connection/environment.go

```diff
@@ -9,15 +9,17 @@ import (
 
 	"github.com/flanksource/commons/logger"
 	"github.com/flanksource/duty/context"
+	"github.com/flanksource/duty/types"
 
 	textTemplate "text/template"
 )
 
 // +kubebuilder:object:generate=true
 type ExecConnections struct {
-	AWS   *AWSConnection   `yaml:"aws,omitempty" json:"aws,omitempty"`
-	GCP   *GCPConnection   `yaml:"gcp,omitempty" json:"gcp,omitempty"`
-	Azure *AzureConnection `yaml:"azure,omitempty" json:"azure,omitempty"`
+	Kubernetes *types.EnvVar    `yaml:"kubernetes,omitempty" json:"kubernetes,omitempty"`
```

Comment:
Might make more sense as a KubernetesConnection object with a kubeconfig: EnvVar - in the future we might have 

```yaml
connections:
   kubernetes:
      eks:
         region: us-east-1
         clusterName: aws-demo
         connection: "connection://default/aws-eks-role"
```